doc/configuring.md

```diff
@@ -0,0 +1,361 @@
+Much of the configuration for genome-grist is about where to find more information about matching genomes.
+
+For Genbank genomes, this is easy! But if you're providing your own genomes and taxonomy information, it's a bit trickier.
+
+## Using Genbank genomes
+
+For Genbank genomes, all the necessary information is available already, or automatically determined by genome-grist!
+
+sourmash already provides pre-built databases containing [all GTDB genomes (R06 rs202)](https://sourmash.readthedocs.io/en/latest/databases.html) as well as [all 700,000 Genbank microbial genomes from July 2020](https://github.com/sourmash-bio/sourmash/issues/1749#issuecomment-947920226).
+
+For genomes available through Genbank (aka with Genbank accessions), genome-grist does the genome retrieval automatically, so you don't need to have them downloaded already.
+
+Taxonomy spreadsheets are available for GTDB (at the databases page) and for Genbank 700k/July 2020 (link upon request).
+
+## Preparing information on private genomes
+
+If you want to use unpublished or private genomes with genome-grist, you'll need to provide your own sourmash database, your own set of genome files and information, and your own taxonomy spreadsheet. Luckily this is all pretty straightforward and we provide tools to help you! Read on!
+
+Note that you can absolutely combine Genbank with your own databases here, or just use your own databases. (If there are overlapping identifiers, the private genomes are chosen first; you might want to do this if you already have a bunch of the Genbank genomes downloaded already, for example.)
+
+### Choosing identifiers for your genomes
+
+You'll need to choose unique identifiers for your genomes. genome-grist requires that your identifier does not have a space, colon (:), or forward slash (/) in it; everything else should be fine.
+
+### Preparing your genome files
+
+You'll need one FASTA file per genome (gzip or bz2 compressed is fine). The filename doesn't matter. It's probably easiest if they're all in one directory, although this isn't necessary.
+
+We suggest naming at the first sequence in each FASTA file with the identifier at the start, space delimited - for example `MY_ID_1.1 first_sequence_name is very special` .
+
+### Creating one or more sourmash databases
+
+You'll need to provide at least one sourmash database for your private collection to genome-grist under the config parameter `private_databases`, which takes a list of paths to sourmash database locations.
+
+Sketch all your genomes with the following command:
+```
+sourmash sketch dna -p k=31,scaled=1000
+```
+
+If you've named your genomes so that the first sequence contains the identifier, you can add `--name-from-first` and then the sequences will be named the right thing for the next step.
+
+If not, you'll need to manually adjust the names of the signatures produced by `sourmash sketch`. (You can do this with `sourmash sig rename`, but there's no simple way to do this in bulk.)
```

Comment:
idk, the `--merge` parameter or `sourmash sig rename` feel like good ways...

Reply:
But those don't work in bulk, right? One... signature... at... a... time... just doesn't scale.

I think what we really need is something like `sourmash sketch fromfile` or a bulk renaming approach or something.

Reply:
err, [sourmash sketch fromfile](https://github.com/sourmash-bio/sourmash/issues/1671#issuecomment-987016199) ref.

Reply:
resolving for now.